src/cellular/encoding/base85.h

```diff
@@ -33,32 +33,33 @@ extern "C" {
 #include <stddef.h>
 
 /** @defgroup base85 Base85 Converter.
-  * Base85 RFC 1924 version. The character set is, in order, 0–9, A–Z, a–z, and
-  * then the 23 characters !#$%&()*+-;<=>?@^_`{|}~.
-  * @{ */
+ * Base85 RFC 1924 version. The character set is, in order, 0–9, A–Z, a–z, and
+ * then the 23 characters !#\$%&()*+-;<=>?@^_`{|}~.
```

Comment:
Please capture the characters in a code block group, e.g.:
```
`!#\$%&()*+-;<=>?@^_`{|}~.`
```
or something similar